junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ThrowableCollector.java

```diff
@@ -170,8 +170,10 @@ public void assertEmpty() {
 	 * {@linkplain TestExecutionResult#failed failed} if it <em>failed</em>
 	 * execution; and {@linkplain TestExecutionResult#successful successful}
 	 * otherwise
+	 * @since 1.6
 	 */
-	TestExecutionResult toTestExecutionResult() {
+	@API(status = MAINTAINED, since = "1.6")
```

Comment:
Maybe it can even be `STABLE`?

Reply:
From [the API Guardian doc]( https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html):

> If `@API` is present on a type, it is considered to hold for all public members of the type as well. However, a member of such an annotated type is allowed to declare a `API.Status` of lower stability.

To me, this suggests that a member is not allowed to declare a `API.Status` of higher stability than the class, which is why I have declared it the same stability as the class-level annotation. If this is not what is intended for the `@API` annotation, then perhaps the documentation for `@API` needs to be tweaked to make this clear.

Reply:
Oh yes you’re right! I didn’t see the level of the class, my bad 😉